internal/engine/compiler/engine.go

```diff
@@ -1368,8 +1334,54 @@ func compileWasmFunction(cmp compiler, ir *wazeroir.CompilationResult) (*code, e
 			err = cmp.compileV128Narrow(o)
 		case wazeroir.OperationV128ITruncSatFromF:
 			err = cmp.compileV128ITruncSatFromF(o)
-		case wazeroir.OperationBuiltinFunctionCheckExitCode:
-			err = cmp.compileBuiltinFunctionCheckExitCode()
+		case wazeroir.OperationNullary:
+			switch op.Kind() {
+			case wazeroir.OperationKindUnreachable:
+				err = cmp.compileUnreachable()
+			case wazeroir.OperationKindMemorySize:
+				err = cmp.compileMemorySize()
+			case wazeroir.OperationKindMemoryGrow:
+				err = cmp.compileMemoryGrow()
+
+			case wazeroir.OperationKindI32WrapFromI64:
+				err = cmp.compileI32WrapFromI64()
+
```

Comment:
spacing in this function is intended, to recall that there are "other ops" that are not handled here